.ci/docker/jruby/test.sh

```diff
@@ -6,8 +6,23 @@ printf '\tTest %-30s %s\n' ${image}
 
 if [ -n "$version" ] ; then
   test_name="Test java -version '$version'"
-  docker run -t --rm $image java -version | grep -q "openjdk version \"$version\|1.$version" && printf '\t\t%-40s %s\n' "${test_name}" "PASSED" || printf '\t\t%-40s %s\n' "${test_name}" "FAILED"
+  docker run -t --rm $image java -version | grep -q "openjdk version \"$version\|1.$version"
+  TEST_JAVA_VERSION_RESULT=$?
+  if [[ $TEST_JAVA_VERSION_RESULT -eq 0 ]]; then
+    printf '\t\t%-40s %s\n' "${test_name}" "PASSED"
+  else
+    printf '\t\t%-40s %s\n' "${test_name}" "FAILED"
+  fi
 fi
 
 test_name="Test Hello World"
-docker run -t --rm $image jruby -e "puts 'Hello World" | grep -q 'Hello World' && printf '\t\t%-40s %s\n' "${test_name}" "PASSED" || printf '\t\t%-40s %s\n' "${test_name}" "FAILED"
+# random operation to verify that ruby evaluates the passed string correctly
+docker run -t --rm $image jruby -e "foo = 3 * 4; puts foo" | grep -q '13'
```

Comment:
```suggestion
docker run -t --rm $image jruby -e "foo = 3 * 4; puts foo" | grep -q '12'
```

Reply:
whoops.. committed my test value. thanks for noticing!

Reply:
https://github.com/elastic/apm-agent-ruby/pull/1367/commits/f0e7c3502323232ca6d51e7aaaa2331c3ce8fd52